packages/backend/src/common/utils/paginationHandler.ts

```diff
@@ -0,0 +1,16 @@
+import { like } from 'drizzle-orm';
+import { PgTableWithColumns } from 'drizzle-orm/pg-core';
+
+export const queryFilters = (entity: PgTableWithColumns<any>, filters: { [key: string]: string | undefined }) => {
```

Comment:
let's just keep the queryfilters in the repository. utils should be generic and not depend on a specific database connector